verifier/src/lib.rs

```diff
@@ -4,6 +4,8 @@
 #![deny(missing_docs, missing_debug_implementations, unsafe_code)]
 #![no_std]
 
+extern crate alloc;
+
```

Comment:
:x: This crate should support [no-std::no-alloc](https://github.com/mobilecoinfoundation/attestation/blob/935c6a18c854cba93ae373c89c426a80ca5136c9/verifier/Cargo.toml#L6)

Suggest removing here and only adding to the test that needs it. Guessing the IDE added this originally.
```suggestion
```

Reply:
Done.